roadmap.md

```diff
@@ -1,68 +1,39 @@
+
+The roadmap below communicates priorities for evolving and extending the scope of WPF for .NET Core through 2020 and into 2021. It will continue to evolve based on market changes and customer feedback, so please note that the plans outlined here are not exhaustive or guaranteed. We welcome your feedback on the roadmap: please feel free to contribute to existing issues or [file a new issue](https://github.com/dotnet/wpf/issues "file a new issue").
+
+### PROPOSED ROADMAP
+| #  | Milestone  |  Target Delivery |
+| :------------: | :------------: | :------------: |
+|1  |Incorpoating .NET Framework servicing fixes  into .NET Core 3.1 and .NET 5 |Ongoing|
+|2  |Ongoing integration with .NET 5 |20H2|
+|3  |Build out Test Infrastructure to add tests to validate and merge community PRs|20H2|
+|4  |Accessibility updates to .NET 5   |21H1|
+|5  |ARM64 Support|21H1|
+
+### Our SLA
+#### TRIAGING GITHUB ISSUES
+We are committed to a 72 hour (3 working days) turn around on triaging and responding to new issues filed on Github starting July 10th.
+Additionally,  Issues filed prior to July 10th are also being triaged. Issues filed in 2020 will be triaged first based on number of reactions/comments, and we expect to complete doing so by the middle of 20H2. Issues filed before 2020 will be triaged after that based on number of reactions/comments.
+
+#### EVALUATING PULL REQUESTS
+In order to enable progress for the community on the WPF project, before the test infrastructure work is completed to enable broader community pull requests, we intend to pick 1-2 code pull requests from the community to manually integrate into the project each month. This is our team's commitment being until our resourcing for WPF ramps-up.
+#### Code PRs considered at this time will be based on:
+•	Recency
+•	Extent of Reactions (Likes, comments)
+•	Fixes to existing issues prioritized over new features
+•	Complexity of the fix
+•	Breadth of impact of the fix
+•	Low potential for regression
+•	Has adequate test coverage
+•	Is not a new feature request
+
+#### About non-code PRs
+We are committed to a 72 hour (3 working days) turn around on triaging and responding to new non-code PRs filed on Github starting July 10th. These are PRs for documentations bugs, documentation enhancements or a general question not related to source code issues/bugs.
+
+### COMMUNITY CALLS
+We will host a WPF segment in the existing [WinUI community calls](https://github.com/microsoft/microsoft-ui-xaml#winui-community-calls "WinUI community calls") once every two months (beginning in July 2020).In these calls we’ll discuss the WPF roadmap, our status and your feedback.
```

Comment:
Shouldn't WPF be merged with the .NET Desktop standups that occur on a monthly basis, and keep WinUI Community Calls focused on WinUI?

Reply:
But if it's owned by the same team as the WinUI and they two Repos are going to work coordinated, it could make sense? Then again whey aren't both meeting one big standup?

Reply:
WPF will be on this month's .NET Desktop standup, as well as the WinUI Community calls.  We expect to do this every couple of months.